resources/views/user/worklog/create.blade.php

```diff
@@ -0,0 +1,80 @@
+@include('templates.header')
+@includeWhen(auth()->user()->is_admin, 'templates.admin-nav')
+@includeUnless(auth()->user()->is_admin, 'templates.nav')
+
+<h2>Create Worklog</h2>
+<form
+    @if(auth()->user()->is_admin)
```

Comment:
try creating a helper file and named it as currentUser which returns auth()->user()

Reply:
i have seen muttiple places where u have retrieve auth()->user() like this so.